addons/website_sale_loyalty/views/website_sale_templates.xml

```diff
@@ -46,13 +46,12 @@
                                             <div class="d-flex flex-row">
                                                 <div class="flex-grow-1">
                                                     <t t-set="program" t-value="reward.program_id"/>
-                                                    <t t-set="points" t-value="website_sale_order._get_real_points_for_coupon(coupon)"/>
+                                                    <t t-set="points" t-value="coupon._format_points(website_sale_order._get_real_points_for_coupon(coupon))"/>
```

Comment:
Don't forget the different points display below:

`<span t-out="points"/> <span t-out="reward.point_name"/>`
`<t t-out="points"/></span> <span t-out="reward.point_name"/>`
`<t t-out="reward.required_points"/></span> <span t-out="reward.point_name"/>`

Reply:
good call